rotkehlchen/externalapis/xratescom.py

```diff
@@ -4,7 +4,7 @@
 from bs4 import BeautifulSoup, SoupStrainer
 
 from rotkehlchen.assets.asset import FiatAsset
-from rotkehlchen.constants.timing import DEFAULT_TIMEOUT_TUPLE
+from rotkehlchen.db import settings
```

Comment:
please don't do this ...these arbitrary import differences are annoying. Be consistent.